### ISSUE TITLE
[Recent Documents] List of recent documents from Apple's `com.apple.recentitems.plist` tend to be outdated

### ISSUE THREAD

**Issue by Jyqft** [ctwise/alfred-workflows#19]
Looking at the workflow's files, it seems to me that the file `get_recent_documents.scpt` is responsible for the population of the recent documents from Apple's side (as opposed to Alfred's); also, 
that same AppleScript file seems to look at `~/Library/Preferences/com.apple.recentitems.plist` for getting the recent documents.

The problem I'm experiencing is that this list of files tends to get outdated. If I compare the results from the workflow (which is based off `com.apple.recentitems.plist`) to ` > Recent Items or to the Dock` or to [the dock stack for recent items](http://osxdaily.com/2011/04/15/add-recent-items-menu-mac-dock/), the latter two always contain the more recently opened files, as opposed to the former. Deleting `com.apple.recentitems.plist` and restarting the Mac seems to fix the update problem, but after that, the .plist file gets frozen in time again.

Are there any possible workarounds for this problem?

**Comment by Jyqft** [comment]
@ctwise: Any updates?

**Comment by MichaelRueegg** [comment]
I experience this as well. Any ideas how to resolve this?

**Comment by ctwise** [comment]
Unfortunately, everything has been deprecated in 10.11. That includes the programmatic way to get this list. So this functionality will probably never be reliable.

**Comment by Jyqft** [comment]
I forgot to mention: I'm on 10.10.5 right now. Even on this version, it's not working reliably. Could've sworn it was working reliably in the past.

I've read about the .plist file being deprecated in 10.11; is there no other alternative file to read from?

**Comment by ctwise** [comment]
No, no alternative. And when I went to do it programatically I found the
API has been deprecated. So any solution would be temporary.
